### DISCUSSION TITLE
May I know where the script for implementing the example of univer sheet is?

### DISCUSSION THREAD

**Discussion by Aminoragit** [dream-num/univer#2901] [Q&A | 常见问题] [answered]
First of all, I want to say thank you for meeting such a wonderful open source library.

I'm studying to use this library, but it's hard to find the code for implementing the example provided by the homepage.
https://univer.ai/examples/sheets

Ofcourse, I know there is a document page for guide and api, but apart from that, I need the code to implement the sample as it is.

Masters, please help beginners.

**Comment by jikkai** [comment, marked as answer]
Most of the code examples can be found under [dream-num/univer.ai](https://github.com/dream-num/univer.ai). You can directly jump to the corresponding code by clicking on the GitHub button shown in the picture below:

<img width="1755" alt="image" src="https://github.com/user-attachments/assets/9dcbbc19-ba7a-4b7f-b145-b5010b92abdc">

**Reply by Aminoragit** [reply]
thx for your help
